packages/CodingStandard/tests/Fixer/Commenting/RemoveEmptyDocBlockFixer/wrong/wrong5.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace SomeNamespace;
+
+    /**
+     * @package foo
+     * @param OutputInterface $output
+     */
```

Comment:
to test indent fixer

Reply:
I think that's realeted to inner indent.

```php
/**
 * @param Type             $sth
 * @param LongerType $sth
 */
```

If not, it needs to be added, since this is the use case responsible for the order

Reply:
No
```
php-cs-fixer describe phpdoc_indent 
Description of phpdoc_indent rule.
Docblocks should have the same indentation as the documented subject.

Fixing examples:
 * Example #1.
   ---------- begin diff ----------
   --- Original
   +++ New
   @@ -1,8 +1,8 @@
    <?php
    class DocBlocks
    {
   -/**
   - * Test constants
   - */
   +    /**
   +     * Test constants
   +     */
        const INDENT = 1;
    }
   
   ----------- end diff -----------
```

Reply:
Then it need to be added